src/time.rs

```diff
@@ -1,11 +1,10 @@
-use std::str::FromStr;
-
 use crate::config::Config;
 use crate::errors::{self, Error};
-use chrono::offset::Utc;
-use chrono::{DateTime, NaiveDate, NaiveDateTime};
+use chrono::{DateTime, NaiveDate, NaiveDateTime, Utc};
 use chrono_tz::Tz;
 use regex::Regex;
+use std::str::FromStr;
+use std::sync::Arc;
```

Comment:
I'm not sure why we are using Arc here, all that we are putting inside is an empty struct and there is no dynamic data to be shared across threads that I can see. Can we just `#[derive(Clone)]
` for the struct?

Reply:
Thx you for pushback I learned a lot here!